graphql_compiler/compiler/sqlalchemy_extensions.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+import sqlalchemy
+
+
+def contains_operator(collection, element):
+    """Return a sqlalchemy BinaryExpression representing this operator.
+
+    Args:
+        - collection: sqlalchemy BindParameter, a collection runtime parameter
```

Comment:
nit: remove the `-` before the parameter names, both here and in the other functions